src/components/TodoList/TodoList.tsx

```
@@ -0,0 +1,187 @@
+              <div className="modal-background has-background-white-ter" />
+              <div className="loader" />
+            </div>
+          </div> */}
+        </section>
+
+        {/* Footer */}
+        {todos.length > 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {todosLeft} items left
+            </span>
+
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={`filter__link ${status === 'all' ? 'selected' : ''}`}
+                data-cy="FilterLinkAll"
+                onClick={() => setStatus('all')}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={`filter__link ${status === 'active' ? 'selected' : ''}`}
+                data-cy="FilterLinkActive"
+                onClick={() => setStatus('active')}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={`filter__link ${status === 'completed' ? 'selected' : ''}`}
+                data-cy="FilterLinkCompleted"
+                onClick={() => setStatus('completed')}
+              >
+                Completed
+              </a>
+            </nav>
+
+            {/* this button should be disabled if there are no completed todos */}
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              data-cy="ClearCompletedButton"
+            >
+              Clear completed
+            </button>
+          </footer>
+        )}
+      </div>
+
+      <div
+        data-cy="ErrorNotification"
+        className={`notification is-danger is-light has-text-weight-normal ${!errorStatus ? 'hidden' : ''}`}
+      >
+        <button data-cy="HideErrorButton" type="button" className="delete" />
```

The 'HideErrorButton' does not have an `onClick` handler to clear the error notification. If the checklist or requirements expect the user to be able to dismiss error messages, you should add an `onClick` handler that calls `setErrorStatus('')` (if available) to hide the error notification.